modules/seqexec/server/src/main/scala/seqexec/server/CaServiceInit.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package seqexec.server
+
+import cats.effect.Sync
+import cats.implicits._
+import edu.gemini.epics.acm.CaService
+import io.chrisdavenport.log4cats.Logger
+import seqexec.model.config.SeqexecEngineConfiguration
+
+object CaServiceInit {
+  // Ensure there is a valid way to init CaService either from
+  // the configuration file or from the environment
+  def caInit[F[_]](conf: SeqexecEngineConfiguration)(implicit F: Sync[F], L: Logger[F]): F[CaService] =
+    L.info("Init EPICS but all subsystems in simulation").unlessA(conf.systemControl.connectEpics) *>
+    (conf.epicsCaAddrList.map(a => F.delay(CaService.setAddressList(a))).getOrElse {
+      F.delay(Option(System.getenv("EPICS_CA_ADDR_LIST"))).flatMap {
+        case Some(_) => F.unit
```

Comment:
@jluhrs I think the original intent was to init CA with `EPICS_CA_ADDR_LIST` if available but we don't do that here, Should we remove this piece of code?

Reply:
Yes, that was the original intent. Can't you just fix it?

Reply:
Ok, I will fix it